src/sentry/static/sentry/app/views/projectDetail/projectTeamAccess.tsx

```diff
@@ -54,54 +38,44 @@ class ProjectTeamAccess extends React.Component<Props, State> {
       );
     }
 
-    const {teams} = project;
-    const {collapsed} = this.state;
-    const canExpand = teams.length > ProjectTeamAccess.MAX_WHEN_COLLAPSED;
-    const teamsToRender =
-      collapsed && canExpand
-        ? teams.slice(0, ProjectTeamAccess.MAX_WHEN_COLLAPSED)
-        : teams;
-    const numberOfCollapsedTeams = teams.length - teamsToRender.length;
-
     return (
-      <React.Fragment>
-        {teamsToRender.map(team => (
+      <Collapsible
+        expandButton={({onExpand, numberOfCollapsedItems}) => (
+          <Button priority="link" onClick={onExpand}>
+            {tn(
+              'Show %s collapsed team',
+              'Show %s collapsed teams',
+              numberOfCollapsedItems
+            )}
+          </Button>
+        )}
+      >
+        {project.teams.map(team => (
           <StyledLink
             to={`/settings/${organization.slug}/teams/${team.slug}/`}
             key={team.slug}
           >
             <IdBadge team={team} hideAvatar />
           </StyledLink>
         ))}
-        {numberOfCollapsedTeams > 0 && (
-          <CollapseToggle priority="link" onClick={this.onCollapseToggle}>
-            {tn(
-              'Show %s collapsed team',
-              'Show %s collapsed teams',
-              numberOfCollapsedTeams
-            )}
-          </CollapseToggle>
-        )}
-        {numberOfCollapsedTeams === 0 && canExpand && (
-          <CollapseToggle priority="link" onClick={this.onCollapseToggle}>
-            {t('Collapse')}
-          </CollapseToggle>
-        )}
-      </React.Fragment>
+      </Collapsible>
     );
   }
 
-  render() {
-    return (
-      <Section>
-        <SectionHeading>{t('Team Access')}</SectionHeading>
+  return (
+    <Section>
+      <SectionHeading>{t('Team Access')}</SectionHeading>
 
-        <div>{this.renderInnerBody()}</div>
-      </Section>
-    );
-  }
+      <div>{renderInnerBody()}</div>
+    </Section>
+  );
 }
 
+ProjectTeamAccess.propTypes = {
```

Comment:
Do we still need propTypes? We're pretty close to the end of the typescript conversion.

Reply:
We still have react/prop-types eslint rule active and it was complaining here.